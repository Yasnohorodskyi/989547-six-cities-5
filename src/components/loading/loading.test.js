import React from "react";
import renderer from "react-test-renderer";

import Loading from "./loading";

it(`Should loading render correctly`, () => {
  const tree = renderer
  .create(<Loading />).toJSON();

  expect(tree).toMatchSnapshot();
});
