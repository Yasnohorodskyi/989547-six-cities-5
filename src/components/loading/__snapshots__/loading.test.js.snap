// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should loading render correctly 1`] = `
<p
  className="status"
>
  Loading
</p>
`;
